ethcore/evm/src/interpreter/gasometer.rs

```diff
@@ -396,7 +396,7 @@ fn calculate_eip1283_sstore_gas<Gas: evm::CostType>(schedule: &Schedule, origina
 		if current == new {
 			// 1. If current value equals new value (this is a no-op), `SSTORE_DIRTY_GAS`
 			// (or if not set, `SLOAD_GAS`) is deducted.
-			schedule.sstore_dirty_gas.unwrap_or(schedule.sload_gas)
+			schedule.sload_gas
```

Comment:
I think the comments need to be updated here and below, it might be easier to just revert #11347

Reply:
I don't think we can revert it entirely because we still need to builtins, no?

Reply:
Good point, we can revert this commit only https://github.com/openethereum/openethereum/pull/11347/commits/10077a79de072dae58c9e49cd89a1702fb45a6cc

Reply:
Not everything in that commit is inaccurate. Maybe I'll just manually rewrite the comments in question.

Reply:
This still needs to be addressed ^^